<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerwidget</class>
 <widget class="QWidget" name="registerwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新会员注册</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namelineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gendercomboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>出生日期</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>联系电话</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phonelineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addresslineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emaillineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>会员类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typecomboBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>月卡</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>季卡</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>年卡</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwlineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
